kube_scheduler/metadata.csv

```diff
@@ -9,6 +9,7 @@ kube_scheduler.gc_duration_seconds.count,gauge,,,,"Number of the GC invocation",
 kube_scheduler.gc_duration_seconds.quantile,gauge,,,second,"GC invocation durations quantiles",0,kube_scheduler,goroutines,
 kube_scheduler.gc_duration_seconds.sum,gauge,,,second,"GC invocation durations sum",0,kube_scheduler,goroutines,
 kube_scheduler.goroutines,gauge,,,,"Number of goroutines that currently exist",0,kube_scheduler,goroutines,
+kube_scheduler.goroutine_by_scheduling_operation,gauge,,,,"Number of running goroutines split by the work they do such as binding (requires k8s v1.26+)",0,kube_scheduler,goroutine_by_scheduling_operation,
```

Comment:
could you also specify that this metrics is in `alpha` stage so they can be removed upstream on the next kubernetes version